packages/blockchain/src/consensus/clique.ts

```diff
@@ -32,6 +32,11 @@ const CLIQUE_SIGNERS_KEY = 'CliqueSigners'
 const CLIQUE_VOTES_KEY = 'CliqueVotes'
 const CLIQUE_BLOCK_SIGNERS_SNAPSHOT_KEY = 'CliqueBlockSignersSnapshot'
 
+// Block difficulty for in-turn signatures
+export const CLIQUE_DIFF_INTURN = BigInt(2)
+// Block difficulty for out-of-turn signatures
+export const CLIQUE_DIFF_NOTURN = BigInt(1)
```

Comment:
Can we change the order of constants and types here (or is it just me who feel that constants would belong to the top?) and then also put all *exported* constants grouped together first?